geode-core/src/main/java/org/apache/geode/management/internal/beans/DistributedSystemBridge.java

```diff
@@ -1405,11 +1405,16 @@ public long getJVMPauses() {
       Map<String, Boolean> senderMap = new HashMap<>();
       for (GatewaySenderMXBean bean : mapOfGatewaySenders.values()) {
         Integer dsId = bean.getRemoteDSId();
-        if (dsId != null) {
-          senderMap.merge(dsId.toString(), bean.isRunning(), Boolean::logicalAnd);
+        if (dsId > -1) {
```

Comment:
would this read better?

if ( dsId == null || dsId <=-1) {
   continue;
}
if(bean.isParallel()){
  senderMap.merge(dsId.toString(), bean.isConnected(), Boolean::logicalAnd);
}
else if(bean.isPrimary(){
  senderMap.put(dsId.toString(), bean.isConnected());
}